main.cpp

```diff
@@ -53,11 +53,11 @@ int main(int argc, char ** argv)
 {
     Logging::Guard guard;
     Logging::Info("Starting JuPedSim - JPScore");
-    Logging::Info(fmt::format("Version {}", JPSCORE_VERSION));
-    Logging::Info(fmt::format("Commit id {}", GIT_COMMIT_HASH));
-    Logging::Info(fmt::format("Commit date {}", GIT_COMMIT_DATE));
-    Logging::Info(fmt::format("Build from branch {}", GIT_BRANCH));
-    Logging::Info(fmt::format("Build with {}({})", compiler_id, compiler_version));
+    Logging::Info(fmt::format(check_fmt("Version {}"), JPSCORE_VERSION));
```

Comment:
why do we have to use `check_fmt`?

Reply:
Thats the compile time check of the format string. Unfortunately we can not hide this inside a function.